mongodb_statedb/__init__.py

```diff
@@ -58,6 +72,10 @@ def exists(self, key: str) -> bool:
         """
         return self._db.find_one({'_id': key}) is not None
 
+    def is_set(self, key: str) -> bool:
```

Comment:
You could also set the value to `None`. To do this properly, it would probably make sense to add a new field to the mongodb entry stored, `isSet` or something similar.

Reply:
I actually rather just delete the entry if we want to reset it so that it doesn't exist.